internal/pubsub/pubsub_test.go

```diff
@@ -185,17 +183,10 @@ func TestConcurrentReceivesGetAllTheMessages(t *testing.T) {
 	defer cancel()
 
 	// Check that all the messages were received.
-	sum := 0
-	for _, n := range receivedMsgs {
-		sum += n
-	}
-	if sum != howManyToSend {
-		t.Errorf("received %d messages, want %d", sum, howManyToSend)
-	}
-	for k, v := range sentMsgs {
-		v2 := receivedMsgs[k]
-		if v2 != v {
-			t.Errorf("got %d for %q, want %d", v2, k, v)
+	for i := 0; i < howManyToSend; i++ {
+		key := fmt.Sprintf("message #%d", i)
+		if _, found := receivedMsgs[key]; !found {
```

Comment:
In this case, I think this is equivalent to
```
if !receivedMsgs[key] {
```

Reply:
Yup; done.